views/authentication/forms/login.hbs

```diff
@@ -80,3 +79,9 @@
 	</div>
 </section>
 <div class="alert-section"></div>
+  <div>
+    {{#if notification.statusCode}}
+   	{{#embed "help/partials/login_failed_hint"}}{{/embed}}
+    {{else}}
```

Comment:
is that 'else' needed here? I'm trying to understand that structure - just curious :)

Reply:
That's right I think I dont need it!